pkg/daemon/ceph/client/pool.go

```diff
@@ -818,3 +793,27 @@ func getCurrentCrushMap(context *clusterd.Context, clusterInfo *ClusterInfo) (Cr
 
 	return crushMap, nil
 }
+
+// GePoolImages returns all the images associated with a pool
+func GePoolImages(context *clusterd.Context, clusterInfo *ClusterInfo, poolName string) ([]string, error) {
+	logger.Debugf("retrieving images for %q pool", poolName)
+
+	// Build command
+	args := []string{"ls", poolName}
+	cmd := NewRBDCommand(context, clusterInfo, args)
+	cmd.JsonOutput = true
+
+	// Run command
+	buf, err := cmd.Run()
+	if err != nil {
+		return nil, errors.Wrapf(err, "failed to images for %q pool. %s", poolName, string(buf))
+	}
+
+	// Unmarshal JSON into Go struct
+	var poolImages []string
+	if err := json.Unmarshal(buf, &poolImages); err != nil {
+		return nil, errors.Wrap(err, "failed to unmarshal mirror pool info response")
```

Comment:
this function is not realted to mirroring, we might need to adjust error message.

Reply:
function is removed now.